lib/virt_autotest/utils.pm

```diff
@@ -943,5 +942,44 @@ sub download_vm_import_disks {
     record_info("All imported disk download is done.");
 }
 
+sub enable_nm_debug {
+    # Enable Network Manager Debug Log Level
+    assert_script_run("nmcli general logging level DEBUG domains ALL", 60);
+    record_info("Enable Network Manager in Debug Level successfully for automation test.");
+}
+
+sub check_activate_network_interface {
+    # Check with activate network interface as required
+    my $target = "openqa.suse.de";
```

Comment:
We may need different target for OSD and O3 (by OPENQA_URL setting).

Reply:
Wow, make sense now. will update to use with OPENQA_URL setting.

Reply:
update to use with OPENQA_URL setting